.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,96 @@
+name: Bug Report
+description: Create a report about an error to help us improve
+title: ""
+labels: ""
+projects: ""
```

Comment:
`projects must be of type String or Array.` [Learn more about error 4.](https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#key-must-be-a-string)